### DISCUSSION TITLE
Can the /umami.js tracking URL use custom domains?

### DISCUSSION THREAD

**Discussion by preshit** [umami-software/umami#773] [Q&A]
Hey community,

Is is possible to install & host umami on a server, but server/embed the tracking code from custom sub-domain for each respective website I want to track? 

For example, can I install umami at `my-analytics-server.com` and track the following websites using a custom domains such as 

- `example.com` where tracking is served from `stats.example.com`
- `another.com` where tracking is served from `stats.another.com`

———

Basically, can the tracking script be linked from a sub-domain of the same website while umami ins installed on a different server?

I've looked at the `data-host-url` in the tracker configuration, but I'm not sure if this is what that's for. I'm also not sure how I can get umami.js hosted on a sub-domain.
